### PR TITLE
Add USDC(wETH) and USDC(wBTC) oracle config

### LINKED ISSUE #41: Prepare oracle deployments
wETH/USDC 91.5% - oracle using chainlink ETH/USD, USD/USDC
wBTC/USDC 86% - oracle using wBTC/BTC, BTC/USD, USD/USDC

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Fixes #41

### CODE REVIEW

File: script/oracles/config/ethereum.json [hunk 1]
```diff
@@ -1,41 +1,28 @@
...
+    "name": "USDC (WBTC)",
+    "collateralToken": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
+    "loanToken": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+    "baseFeed1": "0xfdFD9C85aD200c506Cf9e21F1FD8dd01932FBB23",
+    "baseFeed2": "0xF4030086522a5bEEa4988F8cA5B36dbC97BeE88c",
+    "quoteFeed1": "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6",
+    "quoteFeed2": "0x0000000000000000000000000000000000000000",
+    "vaultConversionSample": 1,
+    "minPrice": 350000000000000000000000000000000000000,
+    "maxPrice": 500000000000000000000000000000000000000,
```
Comment (Rubilmax) [inline script/oracles/config/ethereum.json:25]: if we go above this cap we are rich but the script fails

**Review by Rubilmax — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]

File: script/oracles/config/ethereum.json [hunk 2]
```diff
@@ -1,41 +1,28 @@
...
-        "baseFeed2": "0x0000000000000000000000000000000000000000",
-        "quoteFeed1": "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6",
-        "quoteFeed2": "0x0000000000000000000000000000000000000000",
-        "vaultConversionSample": 1,
-        "minPrice": 100000000000000000000000000,
-        "maxPrice": 150000000000000000000000000,
-        "salt": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    }
+  {
+    "name": "USDC (WETH)",
```
Comment (QGarchery) [inline script/oracles/config/ethereum.json:3]: I thought we were going to use the ETH/USDC chainlink feed, according to [this message](https://morpholabs.slack.com/archives/C02N7CZ088N/p1704965952973829?thread_ts=1704960640.283929&cid=C02N7CZ088N). What changed ?

File: script/oracles/config/ethereum.json [hunk 3]
```diff
@@ -1,41 +1,28 @@
 [
-    {
-        "name": "WETH (wstETH)",
```
Comment (QGarchery) [inline script/oracles/config/ethereum.json:3]: Why removing those configs ?

**Review by QGarchery — CHANGES_REQUESTED** [review]

File: script/oracles/config/ethereum.json (hunk 2 above)
Comment (Rubilmax) [reply script/oracles/config/ethereum.json:3]: @MerlinEgalite

File: script/oracles/config/ethereum.json (hunk 3 above)
Comment (MerlinEgalite) [reply script/oracles/config/ethereum.json:3]: else it will deploy those oracles as well

**Review by Rubilmax — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Comment by Rubilmax** [comment]
Blocked by https://github.com/morpho-org/morpho-blue-oracles/pull/60 then update oracles submodule

**Review by Rubilmax — APPROVED** [review]

**Pull request merged**